README.md

```diff
@@ -1,5 +1,5 @@
 [![Circle CI](https://circleci.com/gh/bitzesty/trade-tariff-backend.svg?style=svg)](https://circleci.com/gh/bitzesty/trade-tariff-backend)
-[![Code Climate](https://codeclimate.com/github/alphagov/trade-tariff-backend/badges/gpa.svg)](https://codeclimate.com/github/alphagov/trade-tariff-backend)
+[![Code Climate](https://codeclimate.com/github/bitzesty/trade-tariff-backend/badges/gpa.svg)](https://codeclimate.com/github/alphagov/trade-tariff-backend)
```

Comment:
realistically alphagov should be the main org that controls this repo for now, even once we move to CF we would be keeping both these repos in sync so, I would revert this change.